### ISSUE TITLE
component throwing exception

### ISSUE THREAD

**Issue by hozefaj** [daviferreira/react-sanfona#72]
Render method throws the below error.

`Invariant Violation: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: undefined. Check the render method of `t`.`

To test it out I am running the exact same code from the docs and still see the issue.

Some investigation revels, that 
`import { Accordion, AccordionItem } from 'react-sanfona'` returns undefined. 
Both `Accordion` and `AccordionItem` are undefined.

Whereas using 
```
import myAccordion from 'react-sanfona'
<myAccordion.Accordion>
  <myAccordion.AccordionItem></myAccordion.AccordionItem>
</myAccordion>
``` 
seem to work fine.

**Comment by psimyn** [comment]
This should be fixed by #63 - I will get npm version updated later today. This will mean your current working example __won't__ work anymore (since default export was removed).

You can use any of these though
```
import { Accordion, AccordionItem } from 'react-sanfona';
import * as myAccordion from 'react-sanfona';
var myAccordion = require('react-sanfona');
```

Thanks for reporting!

**Comment by psimyn** [comment]
@hozefaj I've just published v0.0.12. Please re-open if it's still not working after installing latest version

thanks
